suite-native/discovery/src/discoverySelectors.ts

```diff
@@ -1,13 +1,14 @@
 import { pipe, A, D } from '@mobily/ts-belt';
 
-import { NetworkSymbol } from '@suite-common/wallet-config';
 import {
-    DeviceRootState,
     TokenDefinitionsRootState,
-    selectValidTokensByNetworkSymbolAndDeviceState,
-} from '@suite-common/wallet-core';
+    selectValidTokensByDeviceStateAndNetworkSymbol,
+} from '@suite-common/token-definitions';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { DeviceRootState } from '@suite-common/wallet-core';
 import {
     AccountsRootState,
+    selectAccountsByDeviceStateAndNetworkSymbol,
     selectDeviceAccounts,
 } from '@suite-common/wallet-core/src/accounts/accountsReducer';
```

Comment:
this import is wrong. I know it is not your mistake, but could you fix it aswell please?
```suggestion
} from '@suite-common/wallet-core';
```

Reply:
@PeKne All imports fixed, I unfortunately squashed it